### PR TITLE
Change position of controller image value

### PR DESCRIPTION
**Pull request by cbandy**
This allows for more controllers to be added in the future. The default value is used when upgrading.

Issue: [sc-13977]

### CODE REVIEW

**Review by andrewlecuyer — APPROVED** [review]

File: helm/install/templates/manager.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: operator
-        image: "{{ .Values.image.image }}"
+        image: {{ .Values.controllerImages.cluster | required ".Values.controllerImages.cluster" | quote }}
```
Comment (benjaminjb) [inline helm/install/templates/manager.yaml:20]: When a required value isn't present, does it show just ".Values.controllerImages.cluster" as the error message?

**Review by benjaminjb — APPROVED** [review]
LGTM, just a question about the `required` tag (plus that other comment I had re: required tag)

File: helm/install/templates/manager.yaml (hunk 1 above)
Comment (cbandy) [reply helm/install/templates/manager.yaml:20]: Hmm, it does. `lint` prefixes the message, but not `install`.

```console
$ helm lint --set controllerImages.cluster= helm/install
engine.go:167: [INFO] Missing required value: .Values.controllerImages.cluster

$ helm install --set controllerImages.cluster= … helm/install
Error: INSTALLATION FAILED: execution error at (pgo/templates/manager.yaml:20:53): .Values.controllerImages.cluster
```

Is the following enough?

```suggestion
        image: {{ .Values.controllerImages.cluster | required ".Values.controllerImages.cluster is required" | quote }}
```

```console
$ helm lint --set controllerImages.cluster= helm/install
engine.go:167: [INFO] Missing required value: .Values.controllerImages.cluster is required

$ helm install --set controllerImages.cluster= … helm/install
Error: INSTALLATION FAILED: execution error at (pgo/templates/manager.yaml:20:53): .Values.controllerImages.cluster is required
```

Comment (benjaminjb) [reply helm/install/templates/manager.yaml:20]: Yeah, that looks good to me -- I imagine a lot more errors arise from bad installs

**Pull request merged**